forwarding-rule.tf

```diff
@@ -19,31 +19,37 @@ locals {
 }
 # Assosciate external IP/Port-range to VPN-GW by using Forwarding rules
 resource "google_compute_forwarding_rule" "vpn_esp" {
+  provider    = google-beta
```

Comment:
google_compute_forwarding_rule supports labels in GA. We can upgrade the version constraint to 4.26+
https://github.com/hashicorp/terraform-provider-google/issues/11939